.github/workflows/glucoguide-ci.yaml

```diff
@@ -0,0 +1,74 @@
+name: glucoguide-ci
```

Comment:
Pipeline creation looks good. Looks like the application is building successfully.